src/integrationTest/java/uk/gov/hmcts/divorce/citizen/event/CitizenAddPaymentTest.java

```diff
@@ -55,6 +57,8 @@
 @AutoConfigureMockMvc
 public class CitizenAddPaymentTest {
 
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
```

Comment:
you could use new ObjectMapper().findAndRegisterModules(), then there would be no need for the setUp method below

Reply:
as discussed still getting error after using `new ObjectMapper().findAndRegisterModules(),` hence will keep this as it is